app/src/main/java/org/ole/planet/myplanet/base/BaseContainerFragment.kt

```diff
@@ -69,7 +69,7 @@ abstract class BaseContainerFragment : BaseResourceFragment() {
 
     fun setRatings(`object`: JsonObject?) {
         if (`object` != null) {
-            AdapterCourses.showRating(`object`, rating, timesRated, ratingBar)
+            AdapterCourses.showRating(`object`, rating, timesRated, ratingBar, context)
```

Comment:
I don't think it is necessary to pass context here

Reply:
Okay I will look into this

Reply:
@Okuro3499 I ended up passing in context to use the getString() method. I think the crash is related to something else, so I will be fixing that. Can you let me know if you still think I should not pass context there?